build-system/tasks/check-links.js

```diff
@@ -135,7 +135,12 @@ function filterWhitelistedLinks(markdown) {
   filteredMarkdown = filteredMarkdown.replace(/<code>(.*?)<\/code>/g, '');
 
   // Dailymotion link that is marked dead in Travis CI
-  filteredMarkdown = filteredMarkdown.replace(/https:\/\/developer.dailymotion.com\/player#player-parameters/g, '');
+  filteredMarkdown = filteredMarkdown.replace(
+    /https:\/\/developer.dailymotion.com\/player#player-parameters/g, '');
```

Comment:
Yes, this was an oversight on my part when I first wrote the link checker. Thanks for fixing.